backend/workflow_manager/endpoint/database_utils.py

```diff
@@ -381,25 +386,41 @@ def create_table_query(
             and column definitions.
 
         Note:
-            For 'BIGQUERY', 'SNOWFLAKE', or other database connection classes,
-            the appropriate data types will be mapped based
-            on the Python types of the values in the database_entry dictionary.
-
-            Permanent columns, will always be present in table creation.
+            - A base CREATE SQL query will be created for
+            db-connection class.
+            - Each column name in database_entry will be mapped to
+            db-connection datatype based on thier implementation.
+            - base CREATE SQL query will keep on appending based on
+            db-connection datatype (column key)
+            and database_entry value (column value)
+            - Permanent columns, will always be present in table creation.
```

Comment:
Not mandatory in this PR. It's just a cleanup/refactor
sql_query = conn_cls.get_query(type=create_tabel) or conn_cls.get_create_tabel_query() should be in base connector class
 with default NOne
if it None  use our else